ostree-pulp.sh

```diff
@@ -0,0 +1,405 @@
+#!/bin/bash
+set -euox pipefail
+
+# Provision the software under test.
+./setup.sh
+
+# Get OS data.
+source /etc/os-release
+ARCH=$(uname -m)
+
+# Set up variables.
+TEST_UUID=$(uuidgen)
+IMAGE_KEY="ostree-raw-${TEST_UUID}"
```

Comment:
The ostree-raw does not make sense in this case.

Reply:
fixed.